fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalExcept.java

```diff
@@ -188,6 +188,6 @@ public void computeUniform(Builder builder) {
         for (int i = 0; i < output.size(); i++) {
             replaceMap.put(originalOutputs.get(i), output.get(i));
         }
-        builder.replace(replaceMap);
+        builder.replaceUniformBy(replaceMap);
```

Comment:
why not do same refactor as LogicalIntersect? there are same code in four functions too